build/index.js

```diff
@@ -183,8 +191,13 @@ async function cleanPackage(pkg) {
         await fsPromises.rm(path, { recursive: true, force: true });
 }
 
+async function buildPackage(pkg){
+    const buildCmd = windowsEnvironment ? "win-build" : "build";
+    await npmRun(buildCmd, pkg);
```

Comment:
Roughly the same thoughts as for `clean` -> Except that there's no default behaviour for `build`, so I think the problem is even bigger here :D

It appears what we need for cross platform `build` scripts is some wrapper for `rm -rf`, as well as `cp -r`. I suppose we could always create a _utils_ package that exposes an (or two) executable(s) that do that, if there's no easier way.